### PR TITLE
Import bucket metadata must show CORS and general errors if any are found

### PR DESCRIPTION
**Pull request by marktheunissen**
## Community Contribution License
All community contributions in this pull request are licensed to the project maintainers
under the terms of the [Apache 2 license](https://www.apache.org/licenses/LICENSE-2.0). 
By creating this pull request I represent that I have the right to license the 
contributions to the project maintainers under the Apache 2 license.

## Description

- Fix a bug where any error on the zip file itself (rather than a specific bucket) was not reported, e.g. malformed directory.
- Report whether CORS bucket settings were imported successfully

## How to test this PR?

Example of failed import errors, with this PR:

```
0/2 buckets were imported successfully.
Errors:

Name      : BADFORMAT/corsexporttest/cors.xml
  Error:  malformed zip - expecting format bucket/<config.json>

Name      : corsexporttest
  Notification:  ✔
  Quota:  ✔
  CORS:  ✗
```

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Optimization (provides speedup with no functional changes)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ ] Fixes a regression (If yes, please add `commit-id` or `PR #` here)
- [ ] Unit tests added/updated
- [ ] Internal documentation updated
- [ ] Create a documentation update request [here](https://github.com/minio/docs/issues/new?label=doc-change,title=Doc+Updated+Needed+For+PR+github.com%2fminio%2fmc%2fpull%2fNNNNN)

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -118,4 +118,9 @@ require (
 	google.golang.org/genproto/googleapis/rpc v0.0.0-20240701130421-f6361c86f094 // indirect
 	google.golang.org/grpc v1.65.0 // indirect
 	google.golang.org/protobuf v1.34.2 // indirect
+	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
+
+replace github.com/minio/madmin-go/v3 => github.com/marktheunissen/madmin-go/v3 v3.0.0-20240730061730-2d8255671cb3
+
+replace github.com/minio/pkg/v3 => github.com/marktheunissen/pkg/v3 v3.0.0-20240726014105-5a91ee390421
```
Comment (marktheunissen) [inline go.mod:126]: Remove when dependency is merged

**Review by poornas — APPROVED** [review]

**Review by klauspost — APPROVED** [review]
LGTM 👍🏼

File: go.sum [hunk 2]
```diff
@@ -114,6 +114,10 @@ github.com/lucasb-eyer/go-colorful v1.2.0 h1:1nnpGOrhyZZuNyfu1QjKiUICQ74+3FNCN69
 github.com/lucasb-eyer/go-colorful v1.2.0/go.mod h1:R4dSotOR9KMtayYi1e77YzuveK+i7ruzyGqttikkLy0=
 github.com/lufia/plan9stats v0.0.0-20240513124658-fba389f38bae h1:dIZY4ULFcto4tAFlj1FYZl8ztUZ13bdq+PLY+NOfbyI=
 github.com/lufia/plan9stats v0.0.0-20240513124658-fba389f38bae/go.mod h1:ilwx/Dta8jXAgpFYFvSWEMwxmbWXyiUHkd5FwyKhb5k=
+github.com/marktheunissen/madmin-go/v3 v3.0.0-20240730061730-2d8255671cb3 h1:3m57M9Lq3JauBsFqgpkPEIeDW4+EogeKr3bMT1zJfkM=
```
Comment (harshavardhana) [inline go.sum:117]: Please update these.

**Review by harshavardhana — CHANGES_REQUESTED** [review]

File: go.sum (hunk 2 above)
Comment (marktheunissen) [reply go.sum]: done

**Comment by marktheunissen** [comment]
@harshavardhana please re-review and merge if happy, thanks.

**Review by zveinn — APPROVED** [review]
LGTM

**Pull request merged**